src/Item.js

```diff
@@ -51,6 +52,63 @@ export default class Item {
     return !this._isReady[target.NAME].resolve;
   }
 
+  update(values, source = SOURCE.STATE) {
+    return this._store.schema.setFrom(source, this, values)
+      .then(() => this._synchronize(
+        source === SOURCE.CLIENT_STORAGE ? STATE.EXISTENT : STATE.BEING_UPDATED,
+        source === SOURCE.TRANSPORTER ? STATE.EXISTENT : STATE.BEING_UPDATED
+      )
+    );
+  }
+
+  delete(source = SOURCE.STATE) {
+    return this.remove(source);
+  }
+
+  remove(source) {
```

Comment:
@tchock you are partial right. No default value is needed at all as its done by the switch case anyway. Removed it from item.delete. I also added a small fix in _synchronize (see the very last test in item.spec). Pls review again